modepy/tools.py

```diff
@@ -73,70 +70,82 @@
 
 
 if TYPE_CHECKING:
+    from collections.abc import Sequence
+
+    from numpy.typing import ArrayLike
+
     from modepy.shapes import Shape
     from modepy.typing import ArrayF
 
 
+# NOTE: some places imported gamma from here, but this is deprecated and does
+# not seem to be used anywhere
+gamma = math.gamma
```

Comment:
Can this be removed? I couldn't find any place that imported `gamma` from here and don't recall when that was even the case :\